commands/password-managers/bitwarden/log-in-apikey.template.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Dependency: This script requires the Bitwarden CLI:
+# https://bitwarden.com/help/article/cli/
+#
+# Install via homebrew: `brew install bitwarden-cli`
+
+# These values can be found in the "API Key" section of a web vault:
+# https://vault.bitwarden.com/#/settings/account
+export BW_CLIENTID=""
+export BW_CLIENTSECRET=""
```

Comment:
Do these variables _need_ to be `export`ed? Can you try testing the script where the're set without the `export`? I understand the documentation specifies the option to set these as environment variables, but that's for "normal" usage of the CLI. Given that the `bw login` command is being run within the same context, there may not be a need for the additional leniency (although I'm not sure exactly how Raycast handles environment variables set within script commands, nor am I sure how the `bw` CLI is accessing those variables).

If simply removing the `export`s doesn't work, as another option it may be enough to set them without the `export`s, and also do
```sh
login_token=$(BW_CLIENTID=BW_CLIENTID BW_CLIENTSECRET=BW_CLIENTSECRET bw --raw login --apikey)
```
on L33 below.

Reply:
Great! Please just remove these `export`s, and this should be gtm.
```suggestion
BW_CLIENTID=""
BW_CLIENTSECRET=""
```

Reply:
Thanks for the hint. I did indeed not see, that I forgot to remove this here.
Done now.